docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -65,6 +65,32 @@ storage intact and b) you can build software before you deploy your cluster.
 > is complete. To generate a cost estimate based on your projected usage, use
 > the [pricing calculator](https://cloud.google.com/products/calculator).
 
+<!-- -->
+
+> [!IMPORTANT]
+> Before attempting to execute the following instructions, it is important to
+> consider your project's quota. The `hcls-blueprint.yaml` blueprint creates an
+> autoscaling cluster that, when fully scaled up, can deploy up to 20
+> `a2-highgpu-1g` and `c2-standard-60` VMs.
+>
+> To fully scale up this deployment, the project would require quota for (at least):
```

Comment:
nit: both here and on line 89, I would swap the word deployment for cluster. In my mind the deployment is done once terraform apply has succeeded, it is the cluster that is scaling up.